src/main/java/com/jalasoft/jfc/model/image/ImageConverter.java

```diff
@@ -77,18 +80,21 @@ public FileResponse convert(Param param) throws ConvertException, CommandValueEx
         try {
             commandString = commandImageContext.buildCommand();
 
-            Runtime.getRuntime().exec(commandString);
+            Process process1 = Runtime.getRuntime().exec(commandString);
+            process1.waitFor();
+
 
             if (!commandThumbnailList.isEmpty()) {
                 ContextStrategy commandThumbnailContext = new ContextStrategy(commandThumbnailList);
                 commandString = commandThumbnailContext.buildCommand();
-                Runtime.getRuntime().exec(commandString);
+                Process process2 = Runtime.getRuntime().exec(commandString);
+                process2.waitFor();
             }
+            zipFile(imageParam);
 
             fileResponse.setName(imageParam.getOutputName());
             fileResponse.setStatus(MessageResponse.SUCCESS200.getMessageResponse());
-            fileResponse.setDownload(imageParam.getOutputPathFile()+imageParam.getOutputName());
```

Comment:
Space in plus simbol imageParam.getOutputPathFile()+imageParam.getOutputName()
imageParam.getOutputPathFile() + imageParam.getOutputName()

Reply:
This code line was deleted